src/components/InfoFootnote.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type InfoFootnoteProps = {
+    className?: string;
+    isMobile?: boolean;
+    text: string;
+};
+
+const InfoFootnote = ({ className, isMobile, text }: InfoFootnoteProps) => (
+    <div
+        className={classNames('info-footnote', className, {
+            [`${className}--mobile`]: isMobile,
```

Comment:
when will this styling apply? im not seeing classname passed as a class anywhere

Reply:
classname is initialized with 'info-footnote' now and in future we can reuse with component with a different className (incase we need different style ) , we can just pass the classname into component that will replace 'info-footnote'.